packages/opencensus-exporter-zipkin/src/zipkin.ts

```diff
@@ -138,23 +154,68 @@ export class ZipkinTraceExporter implements Exporter {
    * @param span Span to be translated
    * @param rootSpan Only necessary if the span has rootSpan
    */
-  private translateSpan(span: Span|RootSpan): TranslatedSpan {
+  translateSpan(span: Span|RootSpan): TranslatedSpan {
     const spanTraslated = {
       traceId: span.traceId,
       name: span.name,
       id: span.id,
-      parentId: span.parentSpanId,
-      kind: 'SERVER',
+      // Zipkin API for span kind only accept
+      // (CLIENT|SERVER|PRODUCER|CONSUMER)
```

Comment:
What do PRODUCER and CONSUMER mean?

Reply:
I think these are used in case of messaging calls (ex: Kafka). This indicates that the request is sent to a broker instead of directly to a server and visa versa. Link to old discussion thread https://github.com/census-instrumentation/opencensus-specs/issues/54

Reply:
Makes sense, then, yeah, seems that SERVER is the best we can do assuming it's required.